packages/@aws-cdk/aws-ec2/lib/private/ebs-util.ts

```diff
@@ -24,19 +24,39 @@ function synthesizeBlockDeviceMappings<RT, NDT>(construct: Construct, blockDevic
   return blockDevices.map<RT>(({ deviceName, volume, mappingEnabled }): RT => {
     const { virtualName, ebsDevice: ebs } = volume;
 
+    let finalEbs: any;
```

Comment:
I think we can type this as `CfnLaunchTemplate.EbsProperty | undefined`
```suggestion
    let ebsProperty: CfnLaunchTemplate.EbsProperty | undefined;
```

Reply:
we have to take care of the CfnInstance.EbsProperty also ...

Reply:
I have changed this to:

```let finalEbs:  CfnLaunchTemplate.EbsProperty | CfnInstance.EbsProperty | undefined;```